client/my-sites/customer-home/components/celebrate-launch-modal.jsx

```diff
@@ -19,6 +33,65 @@ function CelebrateLaunchModal( { setModalIsOpen, site } ) {
 		);
 	}, [] );
 
+	function renderUpsellContent() {
+		let contentElement;
+		let buttonText;
+		let buttonHref;
+
+		if ( ! isPaidPlan && ! hasCustomDomain ) {
+			contentElement = (
+				<>
+					<span>
+						{ translate( 'Supercharge your website with a' ) }{ ' ' }
+						<span className="launched__modal-upsell-content-highlight">
+							{ translate( 'custom address' ) }
+						</span>{ ' ' }
+						{ translate( 'that matches your blog, brand, or business.' ) }
```

Comment:
Mixing the string with the span markup will allow translators to translate the string in its entirety, with preserved context. It will also allow RTL language translations to work.
I just made this up from [an example in the docs](https://www.npmjs.com/package/i18n-calypso#mixing-strings-and-markup), but something like this might work?

```suggestion
						{ 
							translate( 
								'Supercharge your website with a {{span}}custom address{{/span}} that matches your blog, brand, or business.',
								{
									components: {
										span: '<span className="launched__modal-upsell-content-highlight" />'
									}
								}
							) 
						}
```

Reply:
Could the span just be a `strong` instead?